packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -49,4 +49,25 @@ export class Debouncer {
       this.reject = reject;
     });
   }
+}
+
+export class I18nService{
+
+  static getTextWithDecorator(text, startTag, endTag) {
+    return `${startTag}${text}${endTag}`;
+  }
+
+  static getTextBold(text) {
+    return I18nService.getTextWithDecorator(text, '<b>', '</b>');
+  }
+
+  static getLabelNameWithCount(labelName, count) {
+    return count > 1 ? `${labelName}_plural` : labelName;
```

Comment:
Technically, zero is plural in english and singular in french. We could have a third variant (for zero/none) for these labels to handle all the pluralization in one place.

Reply:
You mean having another label with a `_zero` suffix? And apply that suffix here following a truthy comparison with 0?

Reply:
That makes sense to me. But so far the zero isn't going to appear I think. I don't think it's possible for a query to return in zero seconds and if a query returns 0 results we'll show the no results component instead of the query summary.

Reply:
If `getLabelNameWithCount` does not handle the zero case, then we'll have to handle it in every place where a pluralized string is used. We'll certainly forget to handle that in some places and it would lead to localization issues (granted those are minor issues). I'm just wondering if having `getLabelNameWithCount` handle this case would make it easier to maintain over time.

Reply:
I agree. I added the zero variants. now we just have to remember to create labels for all 3 scenarios when applicable 👍